Rubberduck.Parsing/Inspections/Resources/InspectionsUI.de.resx

```diff
@@ -876,4 +855,16 @@ Falls der Parameter 'null' sein kann, bitte dieses Auftreten ignorieren. 'null'
   <data name="StopKeywordInspectionName">
     <value>Stop keyword</value>
   </data>
-</root>
+  <data name="ReplaceIfElseWithConditionalStatementQuickFix" xml:space="preserve">
+    <value>Ersetze If/Else mit einer einfachen Zuordnung </value>
+  </data>
+  <data name="BooleanAssignedInIfElseInspectionResultFormat" xml:space="preserve">
+    <value>Boolean Ausdruck '{0}' wurde optional zugewiesen </value>
```

Comment:
"optional" seems problematic here. Instead I'd translate along the lines of "in einer trivialen If/Else-Verzweigung zugewiesen"